app/controllers/api/v1/classrooms_controller.rb

```diff
@@ -3,8 +3,10 @@ class Api::V1::ClassroomsController < Api::V1::BaseController
   before_action :set_classroom, only: %i[show edit update destroy]
 
   def index
-    @classrooms = policy_scope(Classroom)
-    render json: @classrooms.to_json
+    def index
```

Comment:
a `def index` inside a `def index`? :D

Reply:
Some weirdness in a merge conflict resulted in that. fixed it